cluster/kube-up.sh

```diff
@@ -37,10 +37,8 @@ echo "... calling kube-up" >&2
 kube-up
 
 echo "... calling validate-cluster" >&2
-"${KUBE_ROOT}/cluster/validate-cluster.sh"
+validate-cluster
```

Comment:
As the pull request builder just demonstrated, this doesn't quite work. You're not sourcing `kube-util` anywhere here, so `validate-cluster` is just a naked command that doesn't exist yet.

Can you describe what you're trying to do? It almost looks like you're trying to implement virtual functions in bash. That almost works? And it's a method I've considering going down before, I think you have a bit of a longer road ahead of you that involves plumbing every single existing provider (but I might be wrong).